src/request/sign-btc-transaction/SignBtcTransaction.js

```diff
@@ -216,43 +216,82 @@ class SignBtcTransaction {
         // Sort outputs by value ASC, then address ASC
         outputs.sort((a, b) => (a.value - b.value) || (a.address < b.address ? -1 : 1));
 
-        try {
```

Comment:
I believe I removed it because I'm now sure that it will always work and there are now bugs :D

But I see that there are still lots of places where an Error could be thrown, and having a try-catch wrapper doesn't hurt, so I guess I'll re-add it.